libs/istio/docs/README.md

```diff
@@ -0,0 +1,10 @@
+---
+permalink: /
+---
+
+# Jsonnet library for istio
+
+The Jsonnet Kubernetes library is a generated with
```

Comment:
Maybe we can template the readme files as well from Jsonnet? Would make it more maintainable, especially with the versioning.

Reply:
They are really, this is a generated file. :)